### ISSUE TITLE
Issues installing tiny-care-terminal

### ISSUE THREAD

**Issue by leiluspocus** [notwaldorf/tiny-care-terminal#135]
I installed git-standup succesfully but I can't install tiny-care-terminal. I have a kind of infinite loop running :( 

This is the infinite loop I'm having : 
```
gyp WARN EACCES user "root" does not have permission to access the dev dir "/usr/local/lib/node_modules/tiny-care-terminal/node_modules/weak/.node-gyp/8.9.4"
gyp WARN EACCES attempting to reinstall using temporary dev dir "/usr/local/lib/node_modules/tiny-care-terminal/node_modules/weak/.node-gyp"
^C
```

I tried using my regular account but I have an error saying I should run the command again as root.
```
npm ERR! path /usr/local/lib/node_modules/tiny-care-terminal/node_modules/shoe/node_modules/sockjs-client
npm ERR! code EACCES
npm ERR! errno -13
npm ERR! syscall access
npm ERR! Error: EACCES: permission denied, access '/usr/local/lib/node_modules/tiny-care-terminal/node_modules/shoe/node_modules/sockjs-client'
npm ERR!  { Error: EACCES: permission denied, access '/usr/local/lib/node_modules/tiny-care-terminal/node_modules/shoe/node_modules/sockjs-client'
npm ERR!   stack: 'Error: EACCES: permission denied, access \'/usr/local/lib/node_modules/tiny-care-terminal/node_modules/shoe/node_modules/sockjs-client\'',
npm ERR!   errno: -13,
npm ERR!   code: 'EACCES',
npm ERR!   syscall: 'access',
npm ERR!   path: '/usr/local/lib/node_modules/tiny-care-terminal/node_modules/shoe/node_modules/sockjs-client' }
npm ERR!
npm ERR! Please try running this command again as root/Administrator.
```

- **Node Version**: v8.9.4
- **tiny-core-terminal version**: 
```
sh-3.2# npm info tiny-care-terminal | grep version:
  version: '1.3.2',
```

- **Shell**: 
```
GNU bash, version 3.2.57(1)-release (x86_64-apple-darwin17)
Copyright (C) 2007 Free Software Foundation, Inc.
```

- **Terminal Program**: iTerm
- **Operating System**: Mac OSX High Sierra 10.13

Thank you!

**Comment by mojoaxel** [comment]
duplicate of #131

**Comment by leiluspocus** [comment]
To solve this issue, I changed the ownership of my local folder node_modules.  See more details here: http://latrmouh.org/blog/permission-issue-with-npm-while-installing-react-devtools/ 

I deleted the folder tiny-care-terminal in the node_modules. 
```
rm -rf /usr/local/lib/node_modules/tiny-care-terminal/
``` 

I installed it again without the sudo. 
``` 
npm install -g tiny-care-terminal
```

And it worked :) 

Thank you again for this awesome tool!
